seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -691,7 +691,7 @@
       ],
       "tests": [
         "assert(quotient === 2.2, 'message: The variable <code>quotient</code> should equal <code>2.2</code>.');",
-        "assert(/\\//.test(code), 'message: You should use the <code>/</code> operator');"
+        "assert(/4\\.4\\s*\\/\\s*2\\.0/.test(code), 'message: You should use the <code>/</code> operator');"
```

Comment:
The reason I say to have something to count `quotient` variables, although a work around, is that right now, if you have just `var quotient = 2.2;`, this will fail because there is no division sign (which is the second test).

But then if they did something like the original issue has, of 

```js
var quotient = 0.0 / 2.0;
quotient = 2.2;
```

then the test I propose of counting that there is only one `quotient` variable will catch this.

So what I'm proposing is to have like three tests, the current two and another one counting the number of `quotient` variables and the new test would say something like `Change the numbers on the right side of the equal side` or maybe `There should only be one mention of the quotient variable` (this one might be confusing).

Another edge case I thought about is if they did some math like `quotient = 1.1 + 1.1;` after the initial `var quotient = 0.0 / 2.0;` which wouldn't be caught by a regex.

I hope my reasoning makes sense. In sum, I'm trying to catch as many edge cases as possible with any new tests. Let me know if something isn't clear.